packages/framework/get_dependencies.sh

```diff
@@ -3,7 +3,8 @@ ini_file_option=$1
 script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" &> /dev/null && pwd)"
 
 # Data that needs to be updated when GenConfig changes!
-genconfig_sha1=924a08af66f0a0573b5dd1128179731489339aec
+genconfig_sha1=HEAD
```

Comment:
```suggestion
genconfig_sha1=e24b800d6673085850fb612d17ad44744e148464
```

Reply:
Use the SHA1 of the current main branch tip, that way we can upgrade it via PR here and know it's still working.